Learn6_Loop/Learn6_Loop/Learn6_Loop.cpp

```diff
@@ -144,7 +144,7 @@ int main()
 
     //Task 5
     int sum = 0;
```

Comment:
Please create next time new files directly in new branch, so all lines are available here for me to comment

Reply:
if ((i + y) % 2 == 0)
            {
                std::cout << "1";
            }
            else
            {
                std::cout << "0";
            }

std::cout << ((i + y) % 2 == 0);

Reply:
Please read task descriptions more carefully:
while (currentNumberElement <= LastNumberElement)
    {
        float currentElement = firstElement + currentNumberElement * step;
        std::cout << currentElement << " ";
        currentNumberElement++;
    }

basically the last parameter it the number of elements in your sequence, not the last element itself.